content/en/docs/api/build.md

```diff
@@ -139,7 +139,7 @@ metadata:
 spec:
   source:
     url: https://github.com/shipwright-io/sample-go
-    contextDir: docker-build
+    revision: docker-build
```

Comment:
Thanks for spotting that! The text describes setting a specific branch, so the example should match that, but we cannot use `docker-build` for that. Since the `sample-go` only has one branch at the moment (`main`) and I would not like to reference an non-existing example branch, we could either use `main` or `v0.1.0` here. It is worth noting that the revision actually supports branch names, commit SHAs, and tags.

```suggestion
    revision: v0.1.0
```